src/views/RecIM/components/List/Listing/index.jsx

```diff
@@ -338,6 +340,56 @@ const ParticipantListing = ({ participant, minimal, callbackFunction, showPartic
 };
 
 const MatchListing = ({ match, activityID }) => {
+  if (!match) return null;
+  if (match.Team?.length === 2) {
+    return (
+      <ListItem key={match.ID} className={styles.listingWrapper}>
+        <ListItemButton
+          component={Link}
+          to={`/recim/activity/${activityID}/match/${match.ID}`}
+          className={styles.listing}
+        >
+          <Grid container direction="column" alignItems="center">
+            <Grid item container>
+              <Grid item xs={5}>
+                <Typography className={styles.listingTitle}>
+                  {match.Team[0]?.Name ?? <i>TBD</i>}
+                </Typography>
+              </Grid>
+              <Grid item xs={2} textAlign="center">
+                {/* show scores only if match is in the present/past */}
+                {DateTime.now() > DateTime.fromISO(match.Time) ? (
+                  <Typography>
+                    {match.Scores?.find((matchTeam) => matchTeam.TeamID === match.Team[0]?.ID)
+                      ?.TeamScore ?? 'TBD'}{' '}
+                    :{' '}
+                    {match.Scores?.find((matchTeam) => matchTeam.TeamID === match.Team[1]?.ID)
+                      ?.TeamScore ?? 'TBD'}
```

Comment:
```suggestion
                    {match.Scores?.find((matchTeam) => matchTeam.TeamID === match.Team[0]?.ID)
                      ?.TeamScore ?? 0}{' '}
                    :{' '}
                    {match.Scores?.find((matchTeam) => matchTeam.TeamID === match.Team[1]?.ID)
                      ?.TeamScore ?? 0}
```

Scores should be 0 not tbd?